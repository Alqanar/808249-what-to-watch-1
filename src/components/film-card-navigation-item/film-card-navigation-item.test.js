import React from "react";
import renderer from "react-test-renderer";

import FilmCardNavigationItem from "./film-card-navigation-item.tsx";


it(`Film card navigation item correctly renders`, () => {
  const tree = renderer
    .create(<FilmCardNavigationItem
      name="Details"
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
