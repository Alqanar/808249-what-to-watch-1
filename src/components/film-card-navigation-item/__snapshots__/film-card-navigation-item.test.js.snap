// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film card navigation item correctly renders 1`] = `
<li
  className="movie-nav__item"
>
  <a
    className="movie-nav__link"
    href="#"
  >
    Details
  </a>
</li>
`;
